Makefile

```diff
@@ -66,10 +68,20 @@ test-clover:
 	docker build -f examples/clover/Dockerfile .
 
 publish:
-	$(AWS) s3 cp --acl public-read --recursive artifacts/bin/ s3://codeclimate/test-reporter/ --exclude "*" --include "test-reporter-*"
+	$(AWS) s3 cp \
+	  --acl public-read \
+	  --exclude "*" \
+	  --include "test-reporter-$(VERSION)-*" \
+	  --include "test-reporter-latest-*" \
+	  --recursive \
+	  artifacts/bin/ s3://codeclimate/test-reporter/
+
+tag:
+	$(GIT_TAG) --message v$(VERSION) v$(VERSION)
```

Comment:
I think that would be nice to add but I'm wondering if that should go on the release instead of the tag. Might be a good shipbot automation...